index.html

```diff
@@ -159,71 +159,110 @@ <h2>
     </section>
     <section> <h3>Permissions and user prompts</h3>
       <p>
-        The term <a>obtain permission</a>
-        is used for acquiring trust for a given operation.
+        The [[PERMISSIONS]] API provides a uniform way for websites to request
+        permissions from users and query which permissions they have.
       </p>
       <p>
-        The term <dfn>expressed permission</dfn> refers to an act by the user, e.g.
-        via user interface or setting or host device platform features, through which
-        the user approves the permission of a <a>browsing context</a> to access the
-        given functionality.
+        A UA may decide that an explicit user permission
+        is not needed in certain cases, like for certain <a>wake lock types</a>,
+        certain <a>origins</a> or certain <a>top-level browsing contexts</a>.
+        The reasons are unspecified and up to the UA, but could include
+        the site being launched in a standalone app window, or due to
+        pre-configured enterprise policies.
```

Comment:
Likewise here... this sounds ungood... specially if a third-party/enterprise policy can override user setting.

Reply:
I can remove this from the spec, but it is what happens today

Reply:
Can you tell me bit more? Is that in multiple UAs?

Reply:
From Reilly:
> In Chrome's implementation enterprise policies usually override user preferences. I don't think specifications should mandate one behavior or the other in this case.

Reply:
I don't know how other browsers deal with this, or whether they even support enterprise policies, but I assume that Edge might soon follow Chrome's way of doing these things

Reply:
Right. TIL Firefox also does the same: 
https://github.com/mozilla/policy-templates/blob/master/README.md#permissions